deploy/scripts/Create-IomtWebJobsEnvironment.ps1

```diff
@@ -0,0 +1,109 @@
+<#
+.SYNOPSIS
+Creates a new IoMT FHIR Connector for Azure without using Stream Analytics
+.DESCRIPTION
+#>
+param
+(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateLength(5,12)]
+	[ValidateScript({
+        if ("$_" -cmatch "(^([a-z]|\d)+$)") {
+            return $true
+        }
+        else {
+			throw "Environment name must be lowercase and numbers"
+            return $false
+        }
+    })]
+    [string]$EnvironmentName,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('Australia East','East US','East US 2', 'West US', 'West US 2','North Central US','South Central US','Southeast Asia','North Europe','West Europe','UK West','UK South')]
+    [string]$EnvironmentLocation = "North Central US",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('R4')]
+    [string]$FhirVersion = "R4",
+
+    [Parameter(Mandatory = $false)]
+    [string]$SourceRepository = "https://github.com/microsoft/iomt-fhir",
+  
+    [Parameter(Mandatory = $false)]
+    [string]$SourceRevision = "master",
+
+    [Parameter(Mandatory = $true)]
+    [string]$FhirServiceUrl,
+
+    [Parameter(Mandatory = $true)]
+    [string]$FhirServiceAudience,
+
+    [Parameter(Mandatory = $false)]
+    [string]$EnvironmentDeploy = $true
+)
+
+Function BuildPackage() {
+    try {
+        Push-Location $currentPath
+        cd ../../src/console/
+        dotnet restore
+        dotnet build --output $buildPath /p:DeployOnBuild=true /p:DeployTarget=Package
+    } catch {
```

Comment:
nit: I believe that you can just do try/finally in PS (omit the catch) since we aren't taking any action on the error.